src/firebase/remoteConfig/fetchRemoteConfig.ts

```diff
@@ -10,6 +11,14 @@ export function getIsDemoLoginEnabled(): boolean {
   return remoteConfig().getBoolean('isDemoLoginEnabled');
 }
 
+export function getFeatureFlags(): FeatureFlags | undefined {
+  const featureFlags = remoteConfig().getString('featureFlags');
+  if (featureFlags) {
+    return JSON.parse(featureFlags);
+  }
+  return undefined;
```

Comment:
Rather than undefined can we return a default value

Reply:
Not in this case as the default value is the one we have defined in the app elsewhere. Don't want to have multiple sources of truth